javascript/javascript1/week2/FlightBokkingFullNameFunction.js

```diff
@@ -0,0 +1,34 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (class07Students.length >= 6 && studentName !== "Queen" ) {
+        console.log("Cannot add more students to class 07");
+        if (studentName === "Queen" && class07Students.length === 6 && class07Students.includes("Queen")===false) {
+            class07Students.push("Queen");
```

Comment:
includes method returns true or false så you don't need todo class07Students.includes("Queen")===false